src/components/mediaManager/AposMediaManager.vue

```diff
@@ -174,23 +175,23 @@ export default {
 
     // select setters
     select(id) {
-      this.myMedia.map((media) => {
-        media.checkbox.value.data = media.id === id ? ['checked'] : [];
```

Comment:
We're no longer using `checkbox.value.data` for this anymore. I think we might even be able to get rid of `myMedia` altogether.

Reply:
Yeah, there's no indeterminate status. I think we can.

Reply:
I did.

Reply:
No, I think they would be saved to their pieces from the editor.